policyengine_us/variables/gov/states/va/tax/income/rates/va_income_tax_before_credits.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class va_income_tax_before_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VA income tax before credits"
```

Comment:
And in other variables
```suggestion
    label = "Virginia income tax before credits"
```